src/guide/advanced/wasm.md

```diff
@@ -7,7 +7,7 @@ problems with this approach.
 - The ISI syntax is verbose and ugly.
 - The ISI syntax is not familiar for most programmers.
 - While simple ISI smart contracts are compact (usually a few bytes), they
-  need different kinds of be manually optimised.
+  need different kinds of manual optimisations.
 
 In the long run, all of these problems are taken care of by using a
```

Comment:
If this is a long-run mention (ie not actively implemented), shouldn't it be "all these problems will be taken care of?"

Reply:
It's more like a general statement, so I don't think the future tense should be there